### DISCUSSION TITLE
Never received notifications

### DISCUSSION THREAD

**Discussion by rayaneelg** [openwallet-foundation/bifold-wallet#326] [Q&A]
Hello everyone,

We use the public indicio testnet : https://raw.githubusercontent.com/Indicio-tech/indicio-network/main/genesis_files/pool_transactions_testnet_genesis
Using the indicio mediator : http://mediator3.test.indiciotech.io:3000?c_i=eyJAdHlwZSI6ICJkaWQ6c292OkJ6Q2JzTlloTXJqSGlxWkRUVUFTSGc7c3BlYy9jb25uZWN0aW9ucy8xLjAvaW52aXRhdGlvbiIsICJAaWQiOiAiYjE5YTM2ZjctZjhiZi00Mjg2LTg4ZjktODM4ZTIyZDI0ZjQxIiwgInJlY2lwaWVudEtleXMiOiBbIkU5VlhKY1pzaGlYcXFMRXd6R3RtUEpCUnBtMjl4dmJMYVpuWktTU0ZOdkE2Il0sICJzZXJ2aWNlRW5kcG9pbnQiOiAiaHR0cDovL21lZGlhdG9yMy50ZXN0LmluZGljaW90ZWNoLmlvOjMwMDAiLCAibGFiZWwiOiAiSW5kaWNpbyBQdWJsaWMgTWVkaWF0b3IifQ==

We did manage to see the connection promoted to active in our Acapy agent, but the mobile app keep loading after scanning a connection QR code. It never pass that step. 
In the mobile app log we can see the information about the agent .
What do we do wrong ?

**Comment by soubhratra-instnt** [comment]
We are see the same issue. Sometime it works end to end, other times it just keeps spinning indefinitely after the scan with a message appearing that its taking longer and to go back to the home page. Appreciate any insight how to handle this issue.

**Comment by cvarjao** [comment]
is your agent setup to send a Credential Offer or Proof Request right after a successful connection has been stablished (changed to active)? The wallet will wait for a Credential Offer or Proof Request
